### DISCUSSION TITLE
01 - PyTorch Workflow Exercices - Different data set gives values completely off

### DISCUSSION THREAD

**Discussion by gaspardthrl** [mrdbourke/pytorch-deep-learning#615] [Q&A]
Hi when doing the exercises I created a data set of 1000 values with step 1.

I use the same model we used in the course but get absurd losses

However when I create a data set of values from 0 to 10 with step 0.02 it converges fine.

I am using L1Loss and SGD with lr = 1e-3 

Does someone have any idea of what is going on?

**Comment by mrdbourke** [comment]
Hi @gaspardthrl , 

Good question!

You're right, different models/datasets will get different values.

The models used in the course are often tuned (from my own experimentation) to get fairly good results on the given dataset.

So I understand correctly, are you using a different model/dataset? 

Or are you saying your version of the same model/dataset are getting different results?

If you could share your code/where the differences are, that'd be a great help!
